src/core/core.h

```diff
@@ -431,6 +431,25 @@ class CORE_EXPORT Core : public QObject, public Singleton<Core>
         return instance()->_storage->requestMsgsFiltered(user, bufferId, first, last, limit, type, flags);
     }
 
+    //! Request a certain number messages stored in a given buffer, matching certain filters, ascending
+    /** \param buffer   The buffer we request messages from
+     *  \param first    if != -1 return only messages with a MsgId >= first
+     *  \param last     if != -1 return only messages with a MsgId < last
+     *  \param limit    if != -1 limit the returned list to a max of \limit entries
+     *  \param type     The Message::Types that should be returned
```

Comment:
Minor overlooked matter - the `flags` parameter.

```suggestion
     *  \param type     The Message::Types that should be returned
     *  \param flags     The Message::Flags that should be returned
```